Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -59,25 +59,28 @@ def test_font_set(self):
         default_font = idleConf.GetFont(root, 'main', 'EditorWindow')
         default_size = str(default_font[1])
         default_bold = default_font[2] == 'bold'
-        dialog.font_name.set('Test Font')
+        d = dialog
+        d.font_name.set('Test Font')
+        d.font_size.set(default_size)
+        d.font_bold.set(default_bold)
```

Comment:
I was wondering - if you add a similar test as bold_toggle for the font_size_menu selection, maybe the initialization of the 3 Tk variables should be in setUp()?  That would mean that all the tests start with the same values (presumably from GetFont), but the bold_toggle test is a lot different than that now.  font_size 5 in that test isn't even on the menu for size, so it would be an invalid value there.